static/skins/common.css

```diff
@@ -63,7 +63,12 @@ span.label {
     letter-spacing: 0px;
     transform: scale(1.5);
     width: 0.8rem;
+    margin-bottom: 0.7rem;
+}
 
+.rating-stars img.dragon{
+    margin-right: 2rem;
+    margin-left: 1rem;
```

Comment:
idk, I think somewhere between `0` and `0.5rem` looks better in light at least...

<img width="695" alt="image" src="https://user-images.githubusercontent.com/44565/117132013-873a8a80-ada2-11eb-88aa-05f01389e9ed.png">

vs

<img width="735" alt="image" src="https://user-images.githubusercontent.com/44565/117132042-928db600-ada2-11eb-8eb0-2307ae1e492b.png">


vs

<img width="740" alt="image" src="https://user-images.githubusercontent.com/44565/117132078-9cafb480-ada2-11eb-8c71-2e046f42aaea.png">